sections/builder.js

```diff
@@ -1846,10 +1860,18 @@ export default [
         `
       },
       {
-        type: "runnable",
+        type: "text",
+        content: "For MSSQL, triggers on tables can interrupt returning a valid value from the standard insert statements. You can add the `includeTriggerModifications` option to get around this issue. This modifies the SQL so the proper values can be returned. This only modifies the statement if you are using MSSQL, a returning value is specified, and the `includeTriggerModifications` option is set."
```

Comment:
shouldn't it be `delete statements`?

Reply:
btw, what does `returning` for deletion even do? is it supported at all?

Reply:
From my testing, yes, it does work and it is covered in test scenarios already written.